5_People/benefits.md

```diff
@@ -12,16 +12,16 @@ In other words, would something boost your productivity and creativity? Good, fe
 
 ### Allowance Budget
 
-The current monthly allowance budget as of *January 2018* is:
+The current monthly allowance budget as of *January 2020* is:
 
- * Niteans: _€130 ($150)_
- * Founders: _€260_
+ * Niteans: _€130 ($155)_
```

Comment:
USD value is decreased because USD and EUR are closer now than they were a year ago.